interface/src/instruction.rs

```diff
@@ -94,14 +151,47 @@ pub enum StakeInstruction {
     ///
     /// The `u64` is the portion of the stake account balance to be withdrawn,
     /// must be `<= StakeAccount.lamports - staked_lamports`.
-    Withdraw(u64),
+    #[cfg_attr(
+        feature = "codama",
+        codama(account(
+            name = "stake",
+            writable,
+            docs = "Stake account from which to withdraw"
+        )),
+        codama(account(name = "recipient", writable, docs = "Recipient account")),
+        codama(account(name = "clock_sysvar", docs = "Clock sysvar", default_value = sysvar("clock"))),
+        codama(account(
+            name = "stake_history",
+            docs = "Stake history sysvar that carries stake warmup/cooldown history",
+            default_value = sysvar("stake_history")
+        )),
+        codama(account(name = "withdraw_authority", signer, docs = "Withdraw authority")),
+        codama(account(
+            name = "lockup_authority",
+            optional,
+            signer,
+            docs = "Lockup authority, if before lockup expiration"
+        ))
+    )]
+    // backwards compatibility with old IDL demands old `args` name
```

Comment:
Not quite sure what you mean by old IDL, but can we say something like: "Codama requires tuple argument to be named."?

Reply:
I'm referring to people currently using the clients and reference `.args`, `.arg0`, etc. directly from the old Anchor IDL (which codama uses until this PR). L

I'd rather have specific field names here, but it's not worth breaking everyone to do so.

I'll rework the comment

Reply:
Comment clarified in ba555d9